content/includes/footer.html

```diff
@@ -65,12 +65,19 @@
     </div>
   </div>
 </footer>
+
 <!-- End mobile crisis line overlay. See desktop overlay at end of footer -->
-<div id="modal-crisisline" class="va-overlay" role="dialog">
-    <div class="va-crisis-panel">
-      <div class="va-overlay-body va-crisis-panel-body">
-        <h4 class="va-crisis-panel-body-title">Get help from Veterans Crisis Line</h4>
+<div id="modal-crisisline" class="va-overlay va-modal va-modal-large" role="alertdialog">
+    <div class="va-crisis-panel va-modal-inner">
+
+      <h3>Get help from Veterans Crisis Line</h3>
+
+      <button class="va-modal-close va-overlay-close" type="button">
+        <i class="fa fa-close"></i>
+        <span class="usa-sr-only">Close this modal</span>
+      </button>
 
```

Comment:
I'm noticing that if you click on the "x", and not the surrounding button area, that the modal doesn't close because the element that the event is originating doesn't contain the `va-overlay-close` class. The click handler in src/js/legacy/menu.js either needs to be refactored or you can add that `va-overlay-close` to all of the elements in the button.

Reply:
Ahh. i slacked sophia about that problem yesterday but didn't debug as thoroughly as you 😬

Reply:
yeah @ncksllvn making my life easier -- thanks! will adjust.